app/helpers/timelogs_helper.rb

```diff
@@ -0,0 +1,10 @@
+module TimelogsHelper
+  def base_update_string(index)
+    string = "update#{index}"
```

Comment:
no need to have `string =` here.
In ruby the last expression is returned from method.
```
def some_name
  'dsds'
end
```